forte/data/data_store.py

```diff
@@ -489,7 +511,15 @@ def add_group_raw(
             ``tid`` of the entry and its index in the (``type_id``)th list.
 
         """
-        raise NotImplementedError
+        entry = self._new_group(type_name, member_type, tid)
+        try:
+            self.__elements[type_name].add(entry)
+        except KeyError:
+            self.__elements[type_name] = SortedList(key=lambda s: (s[0], s[1]))
```

Comment:
We may also need to update the comparison function for `Group` entry here.

Reply:
Where I can find the order that I can refer to?

Reply:
Previously we use type name as default to compare entries (https://github.com/asyml/forte/blob/master/forte/data/ontology/core.py#L394). Note that some subclasses might overwrite this method (e.g., [Annotation](https://github.com/asyml/forte/blob/master/forte/data/ontology/top.py#L121)).